imports/ui/case-explorer/case-explorer.jsx

```diff
@@ -3,29 +3,45 @@ import { Meteor } from 'meteor/meteor'
 import { connect } from 'react-redux'
 import { createContainer } from 'meteor/react-meteor-data'
 import PropTypes from 'prop-types'
-import { Link, withRouter } from 'react-router-dom'
-import IconButton from 'material-ui/IconButton'
+import { withRouter, Link } from 'react-router-dom'
 import FontIcon from 'material-ui/FontIcon'
 import RaisedButton from 'material-ui/RaisedButton'
-import Cases, { collectionName, isClosed } from '../../api/cases'
+import Cases, { collectionName } from '../../api/cases'
 import { push } from 'react-router-redux'
 import RootAppBar from '../components/root-app-bar'
 import { storeBreadcrumb } from '../general-actions'
-
+import { CaseList, isClosed } from '../case-explorer/case-list'
 import {
-  unitIconsStyle,
-  moreIconColor
+  unitIconsStyle
 } from './case-explorer.mui-styles'
 
+const status = {
+  open: 0,
+  closed: 1,
+  assigned: 2
+}
+
 class CaseExplorer extends Component {
   constructor () {
     super(...arguments)
     this.state = {
       caseId: '',
       expandedUnits: [],
-      unitsDict: {}
+      unitsDict: {},
+      status: 0
+    }
+  }
+
+  handleStatusClicked (value) {
+    if (value === status.open) {
```

Comment:
Since you should change "status" to a boolean, it wouldn't matter, but I just want to point that the contents of this method can be replaced by `this.setState({status:value})` :smile: 
Aim for writing code that is concise and easier to read in the long run :+1: